InterfaceMATLAB/tsgDifferentiate.m

```diff
@@ -0,0 +1,56 @@
+function [result] = tsgDifferentiate(lGrid, mX)
+%
+% [result] = tsgEvaluate(lGrid, points)
```

Comment:
* which method is this? Correct with the above.

Reply:
Fixed.